<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="portGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Port:</string>
    </property>
    <widget class="QTextEdit" name="PortBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ipGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Server IP:</string>
    </property>
    <widget class="QTextEdit" name="IPBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="userGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>User Name:</string>
    </property>
    <widget class="QTextEdit" name="UnameBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="outputGroup">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>321</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QTextEdit" name="outputBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>311</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="inputGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>651</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Input</string>
    </property>
    <widget class="QTextEdit" name="inputBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>641</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="chatButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>93</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Chat!</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>301</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>663</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>startChat()</slot>
 </slots>
</ui>
